controllers/storagecluster/external_resources.go

```diff
@@ -344,12 +348,28 @@ func (r *StorageClusterReconciler) createExternalStorageClusterResources(instanc
 				r.Log.Error(err, "Could not create CephFilesystemSubVolumeGroup.", "CephFilesystemSubVolumeGroup", klog.KRef(found.Namespace, found.Name))
 				return err
 			}
+		case "Namespace":
```

Comment:
This should be `CephBlockPoolRadosNamespace` ? and i don't see this spec passed in the Rook PR.